1st-page/styles/stylesheet.css

```diff
@@ -0,0 +1,115 @@
+.header {
+    color: white;
+    background-color: #003366;
+    padding: 4%;
+    background-image: url("lesson04-mock2-banner-image.jpg");
+    background-repeat: no-repeat;
+    background-size: 150% 50%;
+}
+
+h3 {
+    color: #4CAF50;
+}
+
+.hidden {
+    position: relative;
+    left: -10000px;
+    top: auto;
+}
+
+    .hidden:focus {
+        left: 0;
+    }
+
+nav {
+    background-color: #DCDCDC;
+    padding: 30px;
+}
+
+.preview img {
+    float: right;
+    margin-right: 50px;
+}
+
+@media screen and (max-width: 768px) and (orientation: portrait) {
+    .preview {
+        position: absolute;
```

Comment:
You should not be hiding this using position: absolute. Try toggling visibility instead.